packages/shared/providers.yaml

```diff
@@ -846,6 +846,23 @@ brex-staging:
         base_url: https://platform.staging.brexapis.com
     docs: https://docs.nango.dev/integrations/all/brex
 
+brightcrowd:
+    display_name: BrightCrowd
+    categories:
+        - social
+    auth_mode: OAUTH2_CC
+    token_url: https://bcb-staging.auth.us-east-1.amazoncognito.com/oauth2/token
```

Comment:
we sure about staging?

Reply:
placed it in draft mode so that we can test before pushing, pinged [travis](https://github.com/travistatro) on the other PR, to see if we can get test credentials to test this.

Reply:
@hassan254-prog We should just push this out and get the customer to test this once it is live.